checklists/alerts.md

```diff
@@ -0,0 +1,54 @@
+# Alerts
+
+## Sentry
+
+- [ ] Exceptions that the system may raise - [How to create this alert](https://docs.sentry.io/workflow/notifications/alerts/)
+- [ ] Highly repetitive exceptions within a short time window - [How to create this alert](https://sentry.io/_/resources/customer-success/alert-rules/)
+
+## Papertrail
+
+- [ ] Error keyword - [How to create this alert](https://help.papertrailapp.com/kb/how-it-works/alerts/#create-alert)
+- [ ] Exception keyword - [How to create this alert](https://help.papertrailapp.com/kb/how-it-works/alerts/#create-alert)
+- [ ] Scheduled tasks within a time window (success/failure) - [How to create this alert](https://help.papertrailapp.com/kb/how-it-works/alerts/#inactivity-alerts)
```

Comment:
I don't understand much this: will you alert if tasks succeed as well?

Reply:
Actually, I'm changing into failure or no execution at all